.github/workflows/tests.yml

```
@@ -338,6 +338,25 @@ jobs:
         steps:
             -   name: Checkout code
                 uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6
+                with:
+                    lfs: false
+
+            -   name: Fetch Git LFS objects for governance preflight
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)